### ISSUE TITLE
ColorBox Width for Image Search

### ISSUE THREAD

**Issue by bhsmither** [cubecart/v6#3827]
For the `fmSearch()` function in admin.js, near line 243, the colorbox is specified to be 860 pixels wide.

Yet, in ajax.class.php, `fmSearch()`, near line 337, the function concludes by wrapping the output in a div block with the class of `mail_modal`.

The CSS specifies that the class `mail_modal` has a width of 450 pixels wide.

Switching on and off the width rule for `mail_modal` in the browser's Developer's Tools shows there to be a minimal difference in the display. So the colorbox content is somehow able to escape the confines of an ancestor container.

Was that div block with the class of `mail_modal` intentional? (Or maybe it was unintentional when another function that did need this class on its output was copied to make the new function. I have done that often enough.)

**Comment by abrookbanks** [comment]
From my testing if you remove the 450px width in the browser is makes no change. If you remover it from the CSS and hard refresh its goes to 100% width so it seems to be needed. 

I changed the name from `mail_modal` to `default_modal` as its not used exclusively by the mail test tool. It does seem to be needed at least for padding.
